### PR TITLE
add letter services

### PR DESCRIPTION
**Pull request by rkondratowicz**
screen shot 2018-03-05 at 14 49 22

### CODE REVIEW

File: docs/microservices.json [hunk 1]
```diff
@@ -52,6 +52,44 @@
                 }
             ]
         },
+        {
+            "id": "letters-producer",
```
Comment (nitinprabhu) [inline docs/microservices.json]: We would also need to represent `service bus` I guess ?

File: docs/microservices.json [hunk 2]
```diff
@@ -516,6 +554,11 @@
                     "id": "document-management-store-api-gateway-web",
                     "hard": true,
                     "apis": []
+                },
+                {
+                    "id": "letters-producer",
```
Comment (nitinprabhu) [inline docs/microservices.json]: Is CCD also integration with our API.I thought only CMC is integrating with us at the moment?

File: docs/microservices.json (hunk 1 above)
Comment (rkondratowicz) [reply docs/microservices.json]: I think this is just for applications that we've built

File: docs/microservices.json (hunk 2 above)
Comment (rkondratowicz) [reply docs/microservices.json]: this is cmc

Comment (nitinprabhu) [reply docs/microservices.json]: formatting made it look like it has been added under ccd :(

**Pull request merged**